artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis.cmd

```diff
@@ -58,9 +58,9 @@ call %ARTEMIS_INSTANCE%\etc\artemis.profile.cmd %*
 rem "Create full JVM Args"
 set JVM_ARGS=%JAVA_ARGS%
 if not "%ARTEMIS_CLUSTER_PROPS%"=="" set JVM_ARGS=%JVM_ARGS% %ARTEMIS_CLUSTER_PROPS%
-set JVM_ARGS=%JVM_ARGS% -classpath %ARTEMIS_HOME%\lib\artemis-boot.jar
-set JVM_ARGS=%JVM_ARGS% -Dartemis.home=%ARTEMIS_HOME%
-set JVM_ARGS=%JVM_ARGS% -Dartemis.instance=%ARTEMIS_INSTANCE%
+set JVM_ARGS=%JVM_ARGS% -classpath "%ARTEMIS_HOME%\lib\artemis-boot.jar"
```

Comment:
did you actually test it on windows?


I over tested this on both linux, windows... there's a test under ./scripts validating it.. what is not working?